### PR TITLE
Add flatpak

### PR DESCRIPTION
**Pull request by istori1**

### CODE REVIEW

File: src/data/downloads.tsx [hunk 1]
```diff
@@ -151,6 +151,55 @@ sudo apt install jellyfin`}
...
+      {
+        id: 'flatpak-unstable-button',
+        name: 'Install Instructions',
+        details: (
+          <p>
+            <b>Note:</b> Only stable releases are distributed in Flathub.
+          </p>
+        )
+      }
+    ],
```
Comment (nielsvanvelzen) [inline src/data/downloads.tsx:194]:
```suggestion
    unstableButtons: [],
```

Leave the unstableButtons empty is there is no support for unstable builds.

File: src/data/downloads.tsx [hunk 2]
```diff
@@ -151,6 +151,55 @@ sudo apt install jellyfin`}
...
+    platforms: [Platform.Linux],
+    description: 'Install Jellyfin via Flathub.',
+    stableButtons: [
+      {
+        id: 'flatpak-stable-button',
+        name: 'Install Instructions',
+        details: (
+          <>
+            <p>
+              <b>Note:</b> Make sure Flatpak and Flathub are already setup before installing.
```
Comment (nielsvanvelzen) [inline src/data/downloads.tsx:169]: This note seems a bit out of place, why would someone explicitly opening the flatpak instructions not have flatpak installed/setup?

**Comment by nielsvanvelzen** [comment]
@thornbill right now we have the os types linux, windows, mac and docker. We might want to rename docker to "container" or something so we can add both docker and flatpak in that section. What do you think?

**Comment by thornbill** [comment]
I'm inclined to think it should just be under Linux. 🤷‍♂️ afaik that is the only place where flatpaks are supported unlike docker that can run under other OSes.

**Review by nielsvanvelzen — APPROVED** [review]

File: src/data/downloads.tsx [hunk 3]
```diff
@@ -151,6 +151,42 @@ sudo apt install jellyfin`}
...
+      {
+        id: 'flatpak-stable-button',
+        name: 'Install Instructions',
+        details: (
+          <>
+            <pre>
+              <code>{`flatpak install flathub org.jellyfin.JellyfinServer`}</code>
+            </pre>
+            <p>
+              <b>Note:</b> If you are running on an Intel GPU an adtional extension is required for HDR Tone mapping.
```
Comment (nielsvanvelzen) [inline src/data/downloads.tsx:172]:
```suggestion
              <b>Note:</b> If you are running on an Intel GPU an additional extension is required for HDR Tone mapping.
```

**Review by nielsvanvelzen — APPROVED** [review]

**Pull request merged**